### DISCUSSION TITLE
skin editor's "Big Black Box"

### DISCUSSION THREAD

**Discussion by airi019** [ppy/osu#29422] [Ideas] [answered]
I just found it out on skin editor, this "Big Black Box" add on thing, but it says that is for testing and will be removed soon. I say don't because that's useful for osu!mania players and provides the same functionality as lane cover mod, and without making it unranked.

**Comment by peppy** [comment, marked as answer]
You can drag a black image into the editor or use anything else really.

**Reply by airi019** [reply]
oh what
i didnt knew u could drag files into the skin editor

can u drag gifs too?

**Reply by Tenexxt** [reply]
Also there is already a box element included in argons editor, except you can't Change it's colour yet
